### DISCUSSION TITLE
Short freeze on Safari after select single select custom field

### DISCUSSION THREAD

**Discussion by DFelten** [community/community#61260] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

There is a short freeze on Safari (16.5.2) after selecting a single select custom field.  

We have created story points for our project, and when we make a selection on an issue with Safari, nothing happens on the page for 2 to 3 seconds. The field is then selected. In Chrome, this happens directly.

![CleanShot 2023-07-19 at 14 36 01](https://github.com/community/community/assets/1169185/79dcb766-71fb-4dfc-bed2-50e3f6cd4a63)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by TopherTimeMachine** [comment]
Any progress on this?  I'm having the save issue and would like to see this fixed for safari.
On chrome it works fine.
